Package.swift

```
@@ -19,9 +19,8 @@ let package = Package(
     ],
     exclude: [
         "Source/CarthageIntegration",
-        "Source/CarthageKitTests",
         "Source/Scripts",
-        "Source/XCDBLDTests",
         "Source/carthage/swift-is-crashy.c",
+        "Tests/CarthageKitTests/FakeOldObjc.framework",
```

This is needed to suppress the error below:

```
error: the package has an unsupported layout, modulemap (Carthage/Tests/CarthageKitTests/FakeOldObjc.framework/Modules/module.modulemap) is not allowed to be mixed with sources
fix: move the modulemap inside include directory
```